test-process-active-wraps.js fails on windows

It seems that commit b636ba8186d191c52ee36f2f2b1aebbbb4d95575 introduced the following regression on Windows:
```
>.\Release\node.exe test\simple\test-process-active-
wraps.js

assert.js:98
  throw new assert.AssertionError({
        ^
AssertionError: 1 == 0
    at Immediate._onImmediate (c:\Users\jgilli\dev\node-bis\test\simple\test-pro
cess-active-wraps.js:69:16)
    at processImmediate [as _immediateCallback] (timers.js:340:17)

>
```

@cjihrig Would you mind taking a look?

--- a/test/simple/test-process-active-wraps.js
+++ b/test/simple/test-process-active-wraps.js
@@ -49,7 +49,7 @@ var handles = [];
 
   expect(1, 0);
   var conn = net.createConnection(common.PORT);
-  conn.on('lookup', onlookup); 
+  conn.on('lookup', onlookup);
   conn.on('error', function() { assert(false); });
   expect(2, 1);
   conn.destroy();
@@ -65,8 +65,14 @@ var handles = [];
   });
   function onclose() {
     if (++n === handles.length) {
+      // Allow the server handle a few loop iterations to wind down.
+      // This test is highly dependent on the implementation of handle
+      // closing. If this test breaks in the future, it does not
+      // necessarily mean that Node is broken.
       setImmediate(function() {
-        assert.equal(process._getActiveHandles().length, 0);
+        setImmediate(function() {
+          assert.equal(process._getActiveHandles().length, 0);
+        });
       });
     }
   }

